azure-pipelines/standard.yml

```diff
@@ -18,3 +18,14 @@ steps:
     displayName: 'Install dependencies'
   - script: yarn test
     displayName: 'Test'
+  - script: ./scripts/dist-management/pack-and-upload.sh
+    condition: >
```

Comment:
Showing my yaml ignorance: Is the `>` here just so the subsequent checks can be spread across multiple lines?

Reply:
Also probably worth a comment explaining why we have this `condition`. My interpretation is:
1. Ensure we only copy the built files once (we pick Linux on Node 12)
2. Ensure we only copy for changes to `master` (we pick `Schedule` runs for our `master` branch)

Reply:
> : Is the `>` here just so the subsequent checks can be spread across multiple lines?

Yup

Reply:
Added a comment